scripts/train_fixtures.py

```diff
@@ -66,17 +66,17 @@ def train_fixture_gpu(config_prefix: str) -> None:
             train_fixture_gpu("allennlp/tests/fixtures/srl/")
         else:
             models = [
-                ("basic_classifier", "experiment_seq2seq.jsonnet"),
-                "biaffine_dependency_parser",
-                "constituency_parser",
-                "coref",
-                "decomposable_attention",
+                #("basic_classifier", "experiment_seq2seq.jsonnet"),
+                #"biaffine_dependency_parser",
+                #"constituency_parser",
+                #"coref",
+                #"decomposable_attention",
                 "encoder_decoder/composed_seq2seq",
                 "encoder_decoder/simple_seq2seq",
                 "encoder_decoder/copynet_seq2seq",
-                "esim",
-                "simple_tagger_with_span_f1",
-                "srl",
+                #"esim",
+                #"simple_tagger_with_span_f1",
+                #"srl",
```

Comment:
Are you still working on this? This looks like you're in the middle of things.

Reply:
Thanks for catching this one, this was not an intentional commit.  It's mostly done, just getting CI to pass at this point.